.github/workflows/ci.yml

```diff
@@ -11,18 +11,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        shard_file:
+          - shard.yml
         crystal_version:
           - 0.36.1
           - 1.0.0
         experimental:
           - false
+        include:
+          - shard_file: shard.edge.yml
+            crystal_version: 1.0.0
+            experimental: true
+          - shard_file: shard.yml
+            crystal_version: nightly
+            experimental: true
     runs-on: ubuntu-latest
     continue-on-error: ${{ matrix.experimental }}
     container: crystallang/crystal:${{ matrix.crystal_version }}-alpine
+    env:
+      SKIP_LUCKY_TASK_PRECOMPILATION: "1"
```

Comment:
I'm honestly not sure if this is a Crystal bug or not. When any `postinstall` tasks would run (like for Avram), `SHARDS_OVERRIDE` would **also** impact that shard's environment, and Crystal would complain about `Avram` not having a `shard.edge.yml`.

I don't think we really need to precompile the tasks, though, so this seems like a nice way to speed up the test suite anyways.